src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Module.ts

```diff
@@ -86,6 +88,7 @@ export var register = (angular, config, metaApi) => {
             AdhCrossWindowMessagingModule.moduleName,
             AdhEmbedModule.moduleName,
             AdhMetaApiModule.moduleName,
+            AdhNamesModule.moduleName,
```

Comment:
Some dependencies ware missing. So I will just leave this for now and refector later.